packages/wrangler/src/constellation/deleteProject.ts

```diff
@@ -0,0 +1,52 @@
+import { fetchResult } from "../cfetch";
+import { withConfig } from "../config";
+import { confirm } from "../dialogs";
+import { logger } from "../logger";
+import { requireAuth } from "../user";
+import { Name } from "./options";
+import { constellationBetaWarning, getProjectByName } from "./utils";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../yargs-types";
+import type { Project } from "./types";
+
+export function Options(yargs: CommonYargsArgv) {
+	return Name(yargs)
+		.option("skip-confirmation", {
+			describe: "Skip confirmation",
+			type: "boolean",
+			alias: "y",
+			default: false,
+		})
+		.epilogue(constellationBetaWarning);
+}
+type HandlerOptions = StrictYargsOptionsToInterface<typeof Options>;
+export const Handler = withConfig<HandlerOptions>(
+	async ({ name, skipConfirmation, config }): Promise<void> => {
+		const accountId = await requireAuth(config);
+		logger.log(constellationBetaWarning);
+
+		const proj: Project = await getProjectByName(config, accountId, name);
+
+		logger.log(`About to delete Project '${name}' (${proj.id}).`);
+		if (!skipConfirmation) {
+			const response = await confirm(`Ok to proceed?`);
```

Comment:
This log and confirm would probably fit better as one `confirm` call. Something like `confirm("Are you sure you want to delete project ${name} (${proj.id})? This action cannot be undone.")`

Reply:
We want to print a message even when using `skipConfirmation`, hence the two separate steps.